Proton/Sources/Core/PREditorContentName.h

```diff
@@ -21,25 +21,23 @@
 #ifndef EditorContentName_h
 #define EditorContentName_h
 
-
-#endif /* EditorContentName_h */
-
-#import <Foundation/Foundation.h>
-
 NS_ASSUME_NONNULL_BEGIN
 
-@interface EditorContentName : NSObject
+NS_SWIFT_NAME(EditorContentName)
+@interface PREditorContentName : NSObject
```

Comment:
It is recommended to use 3-letters prefix (unless you are Apple), but here I just followed existed PRTextStorage.

Reply:
I'd like an `NS_SWIFT_NAME` for `TextStorage`.